includes/forms/controllers/class.llms.controller.registration.php

```diff
@@ -51,9 +51,11 @@ public function voucher( $person_id, $data, $screen ) {
 	 */
 	public function register() {
 
-		if ( 'POST' !== strtoupper( getenv( 'REQUEST_METHOD' ) ) || empty( $_POST['action'] ) || 'llms_register_person' !== $_POST['action'] || empty( $_POST['_wpnonce'] ) ) { return; }
+		if ( 'POST' !== strtoupper( getenv( 'REQUEST_METHOD' ) ) || empty( $_POST['action'] ) || 'llms_register_person' !== $_POST['action'] || empty( $_POST['_llms_register_person_nonce'] ) ) { return; }
 
-		wp_verify_nonce( $_POST['_wpnonce'], 'llms_register_person' );
+		if ( ! llms_verify_nonce( $_POST['_llms_register_person_nonce'], 'llms_register_person' ) ) {
```

Comment:
@actual-saurabh `llms_verify_nonce()` usage should be `llms_verify_nonce( '_llms_register_person_nonce', 'llms_register_person' )`

The forms don't work at all as written in this PR